build/python/build/meson.py

```diff
@@ -7,7 +7,7 @@
 def __no_ccache(cmd):
     if cmd.startswith('ccache '):
         cmd = cmd[7:]
-    return cmd
+    return str.strip(cmd)
```

Comment:
What exactly does this fix? Where does the space come from?

Reply:
toolchain.cc and toolchain.cxx are initialized via https://github.com/XCSoar/XCSoar/blob/c8d83bc7be2c97543f2968277a0fc5b10f32d433/build/thirdparty.mk#L41

WRAPPED_CC and WRAPPED_CXX is assigned in https://github.com/XCSoar/XCSoar/blob/c8d83bc7be2c97543f2968277a0fc5b10f32d433/build/compile.mk#L99, when CCACHE is empty we get a leading space, which is removed with this line...

Reply:
I think we should better avoid having the space in the first place, instead of patching the Python script to remove it.

Reply:
I agree, I've adjusted the commit.